Data-Engineering/Airflow/example_ezaf_spark_csv_to_parquet_fts.yaml

```diff
@@ -2,39 +2,39 @@ apiVersion: "sparkoperator.hpe.com/v1beta2"
 kind: SparkApplication
 metadata:
   name: spark-fts-{{ts_nodash|replace("T", "")}}
+  namespace: spark
```

Comment:
Namespace here is not needed, Airflow dynamically set user namespace during DAG execution.

```suggestion
```